Makefile

```diff
@@ -32,6 +34,14 @@ _output/bin/%/checkpoint: cmd/checkpoint/main.go
 	mkdir -p $(dir $@)
 	GOOS=$* go build -o _output/bin/$*/checkpoint github.com/coreos/bootkube/cmd/checkpoint
 
+_output/bin/%/node-agent: cmd/node-agent/main.go pkg/node/*.go
```

Comment:
nit (can be follow-up): using a specific dependency like `pkg/node*.go` seems a little fragile. It would mean if we happened to add another package to the node-agent we would also need to update this. It may make more sense just to use $(GOFILES) even if it is more heavy-handed.

Also, it probably would be beneficial to have these tools be versioned as well - which means we could hypothetically have the same prereqs for all of these tools and combine into single make target (untested):

```
_output/bin/%: LDFLAGS=-X github.com/coreos/bootkube/pkg/version.Version=$(shell $(CURDIR)/build/git-version.sh)
_output/bin/%: $(GOFILES) pkg/asset/internal/templates.go
	mkdir -p $(dir $@)
	GOOS=$* go build -ldflags "$(LDFLAGS)" -o $@ github.com/coreos/bootkube/cmd/$(notdir $@)
```